styles/variables-gallium.less

```diff
@@ -98,6 +98,27 @@
 @agate-item-label-font-size:  30px;
 @agate-item-side-outset:      32px;
 
+// MediaPlayer
+// ---------------------------------------
+@agate-mediacontrols-button-bg-border: transparent;
+@agate-mediacontrols-button-bg-height: 60px;
+@agate-mediacontrols-button-bg-width: 60px;
+@agate-mediacontrols-button-client-width: 60px;
+@agate-mediacontrols-button-height: 60px;
+@agate-mediacontrols-button-icon-padding: 0 30px 0 30px;
+@agate-mediacontrols-button-margin: 0 30px 0 30px;
+@agate-mediacontrols-button-min-width: 60px;
+@agate-mediacontrols-icon-font-size: 60px;
```

Comment:
Maybe the icon font size needs to grow. The current look smaller than GUI guide.
![image](https://user-images.githubusercontent.com/5037429/90703229-2dec0300-e2c8-11ea-99e0-ec028a4ef062.png)

Reply:
I gave it a `font-size: 90px` and for the `playPauseIcon` a `font-size:120px`